### ISSUE TITLE
Cannot use all_flags_state with test data

### ISSUE THREAD

**Issue by eagleonhill** [launchdarkly/python-server-sdk#194]
**Is this a support request?**
This issue tracker is maintained by LaunchDarkly SDK developers and is intended for feedback on the SDK code. If you're not sure whether the problem you are having is specifically related to the SDK, or to the LaunchDarkly service overall, it may be more appropriate to contact the LaunchDarkly support team; they can help to investigate the problem and will consult the SDK team if necessary. You can submit a support request by going [here](https://support.launchdarkly.com/hc/en-us/requests/new) or by emailing support@launchdarkly.com.

Note that issues filed on this issue tracker are publicly accessible. Do not provide any private account information on your issues. If your problem is specific to your account, you should submit a support request as described above.

**Describe the bug**
Cannot use all_flags_state with test data

**To reproduce**
1. Setup a client test data with at least one flag
2. call  all_flags_state  on client

**Expected behavior**
Return all_flags_state

**Logs**
```
  File "ldclient/client.py", line 411, in all_flags_state
    'trackEvents': flag['trackEvents'] or requires_experiment_data,
KeyError: 'trackEvents'
```

**SDK version**
7.5.1

**Comment by keelerm84** [comment]
My apologies for the inconvenience, and thank you for filing this report. I have filed this internally as sc-187612 and will update you here once we have a fix released.

**Comment by keelerm84** [comment]
Fixed in 7.6.1
